### ISSUE TITLE
Product Categories

### ISSUE THREAD

**Issue by Kaschman** [Combine-Labs/schemas#4]

**Comment by cmbirk** [comment]
Schema.org has a `product.category` reference, but it's a singular text field. I propose just using a `product.categories` array

**Comment by Kaschman** [comment]
Hmm do we know if products are and/or should belong to multiple categories?

One concern I have is that if we map product attributes to a category, we could wind up with a weird frankenstein of attributes on a multi-category product.

However, I'm also seeing that perhaps categories fit a tree-like structure so that you belong to a certain category node and belong to all parent nodes as well and inherit parent attributes.

**Comment by cmbirk** [comment]
Yeah, one of the things to think about here also is that this is the top-level schema. The underlying implementation at the database layer may not be a one-to-one. Think of this as what the end-payload of an API would be.

**Comment by cmbirk** [comment]
There is a note in the category schema about category hierarchies, but it uses an informal breadcrumb-like reference
